tools/cnvnator_call.cwl

```diff
@@ -7,19 +7,17 @@ requirements:
   - class: ResourceRequirement
     ramMin: $(inputs.max_memory * 1000)
     coresMin: $(inputs.cpu)
-  - class: InitialWorkDirRequirement
-    listing: $(inputs.input_root)
   - class: DockerRequirement
     dockerPull: 'pgc-images.sbgenomics.com/d3b-bixu/cnvnator:v0.4.1'
 baseCommand: [cnvnator]
 inputs:
   bin_size: { type: 'int', inputBinding: { prefix: '-call'}, doc: "Size of the bins for the task" }
-  input_root: { type: 'File', inputBinding: { prefix: '-root'}, doc: "Input root file" }
+  input_root: { type: 'File', inputBinding: { prefix: '-root' }, doc: "Input root file" }
```

Comment:
So in calculate statistics, CNVnator is actually editing the root file. What I was doing there was copying the file to the working directory then having cnvnator modify it and return it. Here CNVnator is only using the root as an input and returning a separate file.

I can't remember if I mounted the calculate statistics root out of necessity or because its best practice not to edit files without first copying them to the working directory. Here it would just be a waste of time/compute to copy the root.